src/test/java/task_modification_use_case/TaskModificationPresenterTest.java

```diff
@@ -0,0 +1,35 @@
+package task_modification_use_case;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskModificationPresenterTest {
+    //task and event instances (and maps) to be used across tests.
+    private final static TaskModificationOutputData outputData = new TaskModificationOutputData(
+            "Sample Task", 0
+    );
+    private final static String error = "ERROR!";
+
+    @Test
+    public void testPrepareSuccessView(){
+        TaskModificationOutputBoundary outputBoundary = new TaskModificationPresenter();
+        TaskModificationOutputData response = outputBoundary.prepareSuccessView(
+                outputData
+        );
+        assertEquals(outputData.getDayId(), response.getDayId());
+        assertEquals(outputData.getTitle(), response.getTitle());
+    }
```

Comment:
check the assert with 0, response.getDayId() and for the second "Sample Task", response.getTitle()

Reply:
Done. Explicitly comparing actual values of instance attributes against expected.